app/templates/components/events/view/overview/event-apps.hbs

```diff
@@ -1,13 +1,11 @@
 <div class="content">
-  <div class="header">{{t 'Apps'}}</div>
+  <div class="header">{{t 'Website Generator'}}</div>
 </div>
 <div class="content ui stackable grid">
-  <div class="row">
-    <div class="column eight wide center aligned">
-      <button class="ui button">{{t 'Generate Android App'}}</button>
-    </div>
-    <div class="column eight wide center aligned">
-      <a href="{{this.webAppGeneratorUrl}}" target="_blank" class="ui button" rel="noopener noreferrer">{{t 'Generate Web App'}}</a>
-    </div>
+  <div class="row pt-0">
+    {{t 'You would like to have a dedicated event website with all details, sponsors listing, speakers and sessions and the entire schedule? Simply use the website generator. It transfers the data of the event via an API to our website tool and creates a website for you. You can download the entire site as a zip file and upload it to your own web hosting service. No database set up or framework required. This feature is still in Alpha stage, but is already used actively by a number of events. If you press the below button a tab will open where your website is generated.'}}
```

Comment:
Please break the sentences into multiple t statements. This way, translators don't have to retranslate the whole paragraph if a single word changes